editing/add-features/src/main/java/com/esri/samples/add_features/AddFeaturesSample.java

```diff
@@ -66,11 +68,19 @@ public void start(Stage stage) {
       stage.setScene(scene);
       stage.show();
 
-      // create a map with streets basemap
-      ArcGISMap map = new ArcGISMap(Basemap.Type.STREETS, 40, -95, 4);
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
 
-      // create a view for this ArcGISMap
+      // create a map with streets basemap style
```

Comment:
Maybe a 'the' will make this more plain english?
```suggestion
      // create a map with the streets basemap style
```